src/styles/main.scss

```diff
@@ -1,3 +1,61 @@
+@import "./navigation";
+@import "./stars.scss";
+@import "./block.scss";
+@import "./card";
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+p,
+a {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin-bottom: 72px;
+}
+
+.card {
+  flex: 0 1 198px;
+  margin-top: 46px;
+  margin-left: 40px;
+  margin-right: 8px;
+}
+
+.card:nth-child(4),
+.card:nth-child(8) {
+  margin-left: 40px;
+}
+
+@media (min-width: 1200px) {
```

Comment:
You don't need to youse media queries in this task. Just calculate the correct container width.